ui/src/utils/api/models/CommitMeta.ts

```diff
@@ -19,6 +19,12 @@ import { exists, mapValues } from '../runtime';
  * @interface CommitMeta
  */
 export interface CommitMeta {
+    /**
+     * 
```

Comment:
What are these typescript changes?

Reply:
looks like some generated stuff that hasn't been committed with some previous changes @snazy ?

Reply:
I'll revert those and commit them separately

Reply:
will rebase once https://github.com/projectnessie/nessie/pull/1573 is merged